brunch-config.js

```diff
@@ -53,5 +53,8 @@ exports.config = {
 
   npm: {
     enabled: true,
+    globals: {
+      Vue: 'vue/dist/vue.common.js'
```

Comment:
Yeah, there's something messed up with vue though.  The default implementation fails to compile templates, and that was the stackoverflow-recommended fix.  I'll see if I can get around it.